lab/internal/experiment/__init__.py

```diff
@@ -297,11 +297,11 @@ def create_experiment(*,
                       tags: Optional[Set[str]]):
     global _internal
 
-    assert _internal is None
-
-    _internal = Experiment(name=name,
-                           python_file=python_file,
-                           comment=comment,
-                           writers=writers,
-                           ignore_callers=ignore_callers,
-                           tags=tags)
+    if not _internal:
```

Comment:
We should be reinitializing if `_internal` exists, right?

Reply:
ah yes, will change that